docfx_yaml/extension.py

```diff
@@ -26,6 +26,7 @@
 import shutil
 import black
 import logging
+import json
```

Comment:
Nit: import ordering

Reply:
Done. I was able to sort through the first party libraries.